sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/models/SecretProperties.java

```diff
@@ -173,9 +185,9 @@ public OffsetDateTime getExpires() {
      * Set the {@link OffsetDateTime expires} UTC time.
      *
      * @param expires The expiry time to set for the secret.
-     * @return the SecretBase object itself.
+     * @return the SecretProperties object itself.
```

Comment:
the **updated** SecretProperties object.

Reply:
updated